src/families/filecoin/bridge/utils/serializer.ts

```diff
@@ -0,0 +1,77 @@
+import cbor from "cbor";
+import { Transaction } from "../../types";
+import BigNumber from "bignumber.js";
+
+const bigNumberToArray = (v: BigNumber) => {
+  let tmp;
+
+  // Adding byte sign
+  let signByte = "00";
+  if (v.lt(0)) {
+    signByte = "01";
+  }
+
+  if (v.toString() === "") {
+    // to test with null bigint
+    return Buffer.from(signByte, "hex");
```

Comment:
please import from "buffer", because "Buffer" may not be a global in all environments.

Reply:
I am not sure about this change. Could you explain me more about it? I tried to look for buffer lib on the rest of the code but I could not find anything.

Reply:
Ok, it's probably fine for now. On my end, I assume Buffer is a legacy that we'll eventually replace by native arrays (but it's not easy) because it's only really available on Node.
It's not available on web, it's not available in React Native – unless we add a polyfill. Which we did, so it should work! https://github.com/LedgerHQ/ledger-live-mobile/blob/develop/src/polyfill.js#L3

but essentially `import { Buffer } from "buffer"` maybe be needed if we run that code on web.

Reply:
I see. Now I understand it.